.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,20 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.245.0/containers/ubuntu
+{
+	"name": "Ubuntu",
```

Comment:
Was plain Ubuntu sufficient?

Reply:
Ah nevermind. It used the Dockerfile. This is just the name.

Reply:
Yes, ubuntu is the base container.